lib/IDETool/CompilerInvocation.cpp

```diff
@@ -60,6 +60,8 @@ static std::string adjustClangTriple(StringRef TripleStr) {
     OS << "armv5"; break;
   case llvm::Triple::SubArchType::ARMSubArch_v5te:
     OS << "armv5te"; break;
+  case llvm::Triple::SubArchType::ARMSubArch_v4t:
+    OS << "armv4t"; break;
```

Comment:
does this work more generally for armv4? are there other embedded cpus that would be good examples?

Reply:
I suppose I could add armv4, but I don’t have any readily available armv4 hardware that I can test against. It probably works just fine though.